labs/coding-102-rest-python-ga/3.md

```diff
@@ -60,7 +60,7 @@ print (json.dumps(response.json(), indent=4, separators=(',', ': ')))
 Now, look at what the code is doing. Focus on the key code changes.
 
 * `controller='sandboxdnac.cisco.com'`
-    * Assign the variable named `controller` the DNA Center IP or DNS. The DNS assigned here is for the APIC-EM Sandbox.
+    * Assign the variable named `controller` the DNA Center IP or DNS. The DNS assigned here is for the DNA Center Sandbox.
 * `url = "https://" + controller + "/api/v1/ticket"`
```

Comment:
I think it's not /ticket here, we'll have to find the correct url to build here.

Reply:
Okay. Let me know.

Reply:
I think it's `/api/system/v1/auth/token`